tutorial/practicals/LFRic/building_code/1_simple_kernels/LFRic_kernel_structure.md

```diff
@@ -0,0 +1,265 @@
+# LFRic kernel structure
+
+In this part of the tutorial we learn about the structure of an LFRic
```

Comment:
When do users do this part of the tutorial? Can the material here be inserted into the hands-on steps instead?

Reply:
This is supporting material for the first tutorial are using it along the tasks of completing kernels. The READMEs in parts1 and 2 are too long to insert more things, but I rephrased them to clarify this document should be accessible and copied from to complete the tasks.